examples/custom_editable/main.rs

```diff
@@ -8,35 +8,19 @@ use gtk::prelude::*;
 use gtk::subclass::prelude::*;
 use gtk::{gdk, glib};
 
-static CSS: &str = "
-tag {
-    margin: 4px 0px;
-    padding: 4px;
-    border-radius: 4px;
-    background: lightskyblue;
-  }
-  tag box {
-    border-spacing: 4px;
-  }
-  tag label,
-  tag image {
-    color: white;
-  }
-  tag button {
-    min-height: 0;
-    min-width: 0;
-    padding: 0;
-    border: 1px solid white;
-  }
-  
-  entry.tagged {
-    border-spacing: 4px;
-  }
-";
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.editable"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
 
 fn build_ui(application: &gtk::Application) {
     let provider = gtk::CssProvider::new();
```

Comment:
Maybe we could be a bit more correct here and setup the css on connect_startup :-)

Reply:
Fixed